server.tf

```diff
@@ -0,0 +1,299 @@
+    cloudsql_instance_name = "${google_sql_database_instance.master.name}"
+  }
+}
+
+data "template_file" "forseti_server_config" {
+  template = "${local.server_conf}"
+
+  vars {
+    root_resource_id        = "${local.root_resource_id}"
+    gsuite_admin_email      = "${var.gsuite_admin_email}"
+    forseti_email_sender    = "${var.forseti_email_sender}"
+    forseti_email_recipient = "${var.forseti_email_recipient}"
+    storage_bucket_name     = "${local.storage_bucket_name}"
+    sendgrid_api_key        = "${var.sendgrid_api_key}"
+  }
+}
+
+#-------------------#
+# Activate services #
+#-------------------#
+resource "google_project_service" "activate_services" {
+  count              = "${length(local.services_list)}"
+  project            = "${var.project_id}"
+  service            = "${local.services_list[count.index]}"
+  disable_on_destroy = "false"
+}
+
+#-------------------------#
+# Forseti Service Account #
+#-------------------------#
+resource "google_service_account" "forseti_server" {
+  account_id   = "${local.server_sa_name}"
+  project      = "${var.project_id}"
+  display_name = "Forseti Server Service Account"
+}
+
+resource "google_project_iam_member" "project" {
+  count   = "${length(local.server_project_roles)}"
+  role    = "${local.server_project_roles[count.index]}"
+  project = "${var.project_id}"
+  member  = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_read" {
+  count  = "${var.org_id != "" ? length(local.server_read_roles) : 0}"
+  role   = "${local.server_read_roles[count.index]}"
+  org_id = "${var.org_id}"
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "folder_read" {
+  count     = "${var.folder_id != "" ? length(local.server_read_roles) : 0}"
+  role      = "${local.server_read_roles[count.index]}"
+  folder_id = "${var.folder_id}"
+  member    = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_write" {
+  count  = "${var.org_id != "" && var.enable_write ? length(local.server_write_roles) : 0}"
+  role   = "${local.server_write_roles[count.index]}"
+  org_id = "${var.org_id}"
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "folder_write" {
+  count     = "${var.folder_id != "" && var.enable_write ? length(local.server_write_roles) : 0}"
+  role      = "${local.server_write_roles[count.index]}"
+  folder_id = "${var.folder_id}"
+  member    = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+/*******************************************
+  Forseti Firewall Rules
+ *******************************************/
+resource "google_compute_firewall" "forseti-server-deny-all" {
+  name                    = "forseti-server-deny-all-${local.random_hash}"
+  project                 = "${var.project_id}"
```

Comment:
If using shared vpc, this would need to be the host project id:
`  project                 = "${var.vpc_host_project_id == "" ? var.project_id : var.vpc_host_project_id}"`